var/spack/repos/builtin/packages/py-astor/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class PyAstor(PythonPackage):
+    """
+    astor is designed to allow easy manipulation of Python source via the AST.
+    """
+
+    homepage = "https://pypi.python.org/pypi/astor"
+    url      = "https://pypi.io/packages/source/a/astor/astor-0.8.0.tar.gz"
+
+    version('0.8.0', '37a6eed8b371f1228db08234ed7f6cfdc7817a3ed3824797e20cbb11dc2a7862')
+    version('0.6', '175ec395cde36aa0178c5a3120d03954c65d1ef4bb19ec4aa30e9d7a7cc426c4')
```

Comment:
@adamjstewart Style question: should this be `sha256=...` or is it OK to just put in the checksum without specifying `sha256`?

Reply:
This is a style question. Last I recall, @tgamblin prefers using `sha256=` so that it's more clear what that super long string of random characters after the version number is. Technically, it works with or without it.